server/routes/todoListRouter.js

```diff
@@ -32,13 +32,13 @@ todoListRouter.post('/category', async (req, res) => {
 });
 
 todoListRouter.post('/item', async (req, res) => {
-  var title = req.body.title || 'opoipoi with E';
-  var description =  req.body.description || 'oipoipiwith E';
-  var duration =  req.body.duration || 3600;
-  var start =  req.body.start || '2022-04-20T10:30:20';
-  var end_date =  req.body.end || '2022-04-20T11:30:20';
-  var in_calendar =  req.body.inCalendar || true;
-  var categoryId =  req.body.categoryId || 13;
+  var title = req.body.title;
+  var description =  req.body.description;
+  var duration =  req.body.duration;
+  var start =  req.body.start;
+  var end_date =  req.body.end;
+  var in_calendar =  req.body.inCalendar;
```

Comment:
Wouldn't it be req.body.in_calendar here as well ? Object from the front-end should have matching properties to PG table names.